### ISSUE TITLE
Selectize Group Vertical Alignment

### ISSUE THREAD

**Issue by Semafoor** [dreamRs/shinyWidgets#104]
Add an option in the Selectize Group UI to allow for vertical alignment of the selectize inputfields.

**Comment by pvictor** [comment]
Hello,
You can use argument `inline` in `selectizeGroup` for vertical layout : 

```r
library(shiny)
library(shinyWidgets)

data("mpg", package = "ggplot2")

ui <- fluidPage(
  fluidRow(
    column(
      width = 4, 
      selectizeGroupUI(
        id = "my-filters", 
        label = "Filters:",
        inline = FALSE,  ### <<<<<---------------- Add this to use vertical layout
        params = list(
          manufacturer = list(inputId = "manufacturer", title = "Manufacturer:"),
          model = list(inputId = "model", title = "Model:"),
          trans = list(inputId = "trans", title = "Trans:"),
          class = list(inputId = "class", title = "Class:")
        )
      )
    ),
    column(
      width = 8,
      dataTableOutput(outputId = "table")
    )
  )
)

server <- function(input, output, session) {
  res_mod <- callModule(
    module = selectizeGroupServer,
    id = "my-filters",
    data = mpg,
    vars = c("manufacturer", "model", "trans", "class")
  )
  output$table <- renderDataTable(res_mod())
}

shinyApp(ui, server)
```

I'll do the same for `pickerGroup`.

Victor

**Comment by Semafoor** [comment]
Wonderful! Truly amazing that you've added the functionality that rapidly.

**Comment by Semafoor** [comment]
Stupid question probably, but how do I get the new functionality in R? I've tried updating the shinywidgets package, but it return unused argument inline = FALSE..

**Comment by pvictor** [comment]
You're welcome :)
No stupid question, you need to install package from GitHub, I should have added these lines : 

```
# install.packages("devtools")
devtools::install_github("dreamRs/shinyWidgets")
```

Victor

**Comment by Semafoor** [comment]
Brilliant! It works like a charm! 

Is there any way to change the way the selectors are visualized? I.e. a select all button, just like the pickerinput?

**Comment by stephLH** [comment]
Hi Victor,
Is there any news about integrating the `inline` argument to the `PickerGroup` function ?

**Comment by sylvaine31** [comment]
> 
> 
> Hi Victor,
> Is there any news about integrating the `inline` argument to the `PickerGroup` function ?

Hi Victor, 
I use your filter group functions in many apps, thanks a lot four your job!
I agree that the inline argument  in the `PickerGroup` function would be useful! 
it will be also perfect if the `multiple = FALSE` could be also available in both `selectizeGroup` and `pickerGroup` ... :)

**Comment by pvictor** [comment]
Hi,
`pickerGroup` also have an `inline` argument now. Please install dev version to try it out : 

```r
devtools::install_github("dreamRs/shinyWidgets")
```

I close this issue for now, re-open it if you encouter some trouble. For the `multiple` argument I don't think there's a need for `pickerGroup`, please open a new issue to describe the functionnality if you want.

Victor

**Comment by sylvaine31** [comment]
Merci Victor,
J'ai voulu aller te voir après ton talk à la USER, mais tu étais cerné... Merci encore pour ShinyWidget, c'est vraiment super utile!

Sylvaine


________________________________
De : Victor Perrier <notifications@github.com>
Envoyé : mercredi 17 juillet 2019 15:13
À : dreamRs/shinyWidgets
Cc : sylvaine31; Comment
Objet : Re: [dreamRs/shinyWidgets] Selectize Group Vertical Alignment (#104)


Hi,
pickerGroup also have an inline argument now. Please install dev version to try it out :

devtools::install_github("dreamRs/shinyWidgets")

I close this issue for now, re-open it if you encouter some trouble. For the multiple argument I don't think there's a need for pickerGroup, please open a new issue to describe the functionnality if you want.

Victor

**Comment by pvictor** [comment]
De rien! C'est gentil, oui il y avait du monde effectivement !

**Comment by michaelgrund** [comment]
As @MikeCilliers pointed out ``inline = FALSE`` does not result in vertical alignment. Instead the input fields are aligned somehow horizontally, see below.
![screen_shiny](https://user-images.githubusercontent.com/23025878/157001674-b9ab6d6b-3dfb-4060-a276-cdbe70741621.png)

**Comment by MikeCilliers** [comment]
I have opened a new issue as this one is closed, see
https://github.com/dreamRs/shinyWidgets/issues/481

**Comment by michaelgrund** [comment]
> I have opened a new issue as this one is closed, see #481

Already opened one just a few our earlier than you (see #480) @MikeCilliers  :stuck_out_tongue_winking_eye:.
